tests/e2e-test-framework/framework/utils.py

```diff
@@ -508,7 +574,113 @@ def annotate_pvc(
+
+        Args:
+            namespace (str): The namespace of the custom objects to be cleared.
+
+        Returns:
+            None: This function does not return anything.
+        """
+        try:
+            self.custom_objects_api.delete_collection_namespaced_custom_object(
+                group=const.CR_GROUP,
+                version=const.CR_VERSION,
+                namespace=namespace,
+                plural=const.VOLUMES_PLURAL,
+                grace_period_seconds=0,
+                propagation_policy="Foreground",
+            )
+            logging.info("CR volumes: delete request sent")
+            for plural in [
+                const.DRIVES_PLURAL,
+                const.AC_PLURAL,
+                const.ACR_PLURAL,
+                const.LVG_PLURAL,
+            ]:
+                self.custom_objects_api.delete_collection_cluster_custom_object(
+                    group=const.CR_GROUP,
+                    version=const.CR_VERSION,
+                    plural=plural,
+                    grace_period_seconds=0,
+                    propagation_policy="Foreground",
+                )
+                logging.info(f"CR {plural}: delete request sent")
+            self.core_v1_api.delete_collection_namespaced_persistent_volume_claim(
+                namespace=namespace
+            )
+            logging.info("waiting for resources to be in the removing state")
+            time.sleep(10)
+            lvg_list = self.custom_objects_api.list_cluster_custom_object(
+                group=const.CR_GROUP,
+                version=const.CR_VERSION,
+                plural="logicalvolumegroups",
+            )["items"]
+            for lvg in lvg_list:
+                if "finalizers" in lvg.get("metadata", {}):
+                    lvg["metadata"]["finalizers"] = []
+                    self.custom_objects_api.replace_cluster_custom_object(
+                        group=const.CR_GROUP,
+                        version=const.CR_VERSION,
+                        namespace=namespace,
+                        plural=const.LVG_PLURAL,
+                        name=lvg["metadata"]["name"],
+                        body=lvg,
+                    )
+            for v in self.list_volumes():
+                if "finalizers" in v.get("metadata", {}):
+                    v["metadata"]["finalizers"] = []
+                    self.custom_objects_api.replace_namespaced_custom_object(
+                        const.CR_GROUP,
+                        const.CR_VERSION,
+                        namespace,
+                        plural=const.VOLUMES_PLURAL,
+                        name=v["metadata"]["name"],
+                        body=v,
+                    )
+        except ApiException as e:
+            print(
+                f"Exception when calling CustomObjectsApi->delete_namespaced_custom_object: {e}"
+            )
+
+    def recreate_pod(self, pod: V1Pod) -> V1Pod:
```

Comment:
do we need to parse the whole pod, maybe name and namespace is enough?

Reply:
makes sense, thank you